### PR TITLE
fix!: Change default from `--release` to `--debug`

### LINKED ISSUE #31: Make debug builds the default
For
- Easier debugging
- Faster builds

### PR DESCRIPTION
**Pull request by GitaWei**
Fixes #31

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -46,7 +46,6 @@ struct Args {
     clear_cache: bool,
     debug: bool,
     dep: Vec<String>,
-    extern_: Vec<String>,
```
Comment (epage) [inline src/main.rs:49]: How come this commit is showing up?

File: src/main.rs [hunk 2]
```diff
@@ -142,9 +142,16 @@ fn parse_args() -> Args {
...
+            .arg(Arg::new("release")
+                .help("Build a release executable, an optimised one.")
+                .short('r')
+                .long("release")
+                .conflicts_with_all(&["debug", "bench"])
+            )
             .arg(Arg::new("debug")
                 .help("Build a debug executable, not an optimised one.")
                 .long("debug")
+                .conflicts_with_all(&["release"])
```
Comment (epage) [inline src/main.rs:154]: How come we are leaving this behind?  It looks like cargo commands usually only expose one or the other

File: src/main.rs (hunk 1 above)
Comment (GitaWei) [reply src/main.rs:49]: This was a potential fix for making debug builds the default MO per #31 .  It did not seem like using the --debug flag to turn debugging off was correct. Apologies if this was out of spec.

File: src/main.rs (hunk 2 above)
Comment (GitaWei) [reply src/main.rs:154]: Sorry, leaving which behind?

Comment (epage) [reply src/main.rs:154]: Yes, let's remove `--debug`.

**Pull request closed without merging**